livekit-agents/livekit/agents/llm/utils.py

```diff
@@ -109,27 +110,44 @@ def is_context_type(ty: type) -> bool:
 
 @dataclass
 class SerializedImage:
-    data_bytes: bytes
-    media_type: str
     inference_detail: str
+    media_type: str | None
```

Comment:
how does media_type differ from a mime_type? 

If it doesn't, let's just call it mime_type here as well for consistency?